opencti-platform/opencti-front/src/private/components/settings/Settings.jsx

```diff
@@ -428,28 +428,31 @@ const Settings = () => {
                     <Typography variant="h4" gutterBottom={true} stye={{ float: 'left' }}>
                       {t_i18n('OpenCTI platform')}
                     </Typography>
-                    <div style={{ float: 'right', marginTop: isSensitiveModificationEnabled ? theme.spacing(-5) : theme.spacing(-4.5), position: 'relative' }}>
+                    <div style={{ float: 'right', marginTop: isSensitive ? theme.spacing(-5) : theme.spacing(-4.5), position: 'relative' }}>
                       {!isEnterpriseEdition ? (
                         <EnterpriseEditionButton disabled={!isAllowed} inLine />
                       ) : (
                         <DangerZoneBlock
+                          type={'ce_ee_toggle'}
                           sx={{
                             root: { border: 'none', padding: 0, margin: 0 },
                             title: { position: 'absolute', zIndex: 2, left: 4, top: 9, fontSize: 8 },
                           }}
                         >
-                          {({ disabled }) => (
+                          {() => (
                             <>
                               <Button
                                 size="small"
                                 variant="outlined"
-                                color={isSensitiveModificationEnabled ? 'dangerZone' : 'primary'}
+                                color={isSensitive ? 'dangerZone' : 'primary'}
                                 onClick={() => setOpenEEChanges(true)}
-                                disabled={disabled}
-                                style={isSensitiveModificationEnabled ? {
-                                  color: theme.palette.dangerZone.text.primary,
-                                  borderColor: theme.palette.dangerZone.main,
-                                } : undefined}
+                                disabled={isSensitive && !isAllowed}
```

Comment:
I would probably revert it back to use the disabled information given by the DangerZoneBlock

Reply:
done